cis_enforcements/aws/cis_v300_s5_networking/security_group.tf

```diff
@@ -0,0 +1,70 @@
+# AWS > VPC > Security Group > Ingress Rules > Approved
+resource "turbot_policy_setting" "aws_vpc_security_group_ingress_rules_approved" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/securityGroupIngressRulesApproved"
+  value    = "Check: Approved"
+  # value    = "Enforce: Delete unapproved"
+  note     = "AWS CIS v3.0.0 - Controls: 5.2, 5.3, 5.4"
+}
+
+# AWS > VPC > Security Group > Ingress Rules > Approved > Rules
+resource "turbot_policy_setting" "aws_vpc_security_group_ingress_rules_approved_rules" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/securityGroupIngressRulesApprovedRules"
+  note     = "AWS CIS v3.0.0 - Controls: 5.2, 5.3, 5.4"
+  template_input = <<-EOT
+    {
+      value: constant(value: "REJECT *")
```

Comment:
no reason to do this as a constant.  only if we have separate enforce/check options in the policy setting.  in this case the calc policy will be the same regardless of check/enforce.